ScriptingEngineTester/src/main/resources/zencode_tests/expressions/optional_asstring_long_1.zc

```diff
@@ -0,0 +1,4 @@
+#output: 4711
+
+var x = 4711 as long?;
+println(x);
```

Comment:
Currently broken: It tries to convert the `long?` to string without unwrapping it first...

This is compiled as:
```asm
   L0
    LINENUMBER 0 L0
    LINENUMBER 3 L0
    LDC 4711
    INVOKESTATIC java/lang/Long.valueOf (J)Ljava/lang/Long;
    ASTORE 1
   L1
    LINENUMBER 4 L1
    ALOAD 1
   L2
    INVOKESTATIC java/lang/Long.toString (J)Ljava/lang/String;
    INVOKESTATIC org/openzen/zenscript/scriptingexample/tests/SharedGlobals.println (Ljava/lang/String;)V
    RETURN
    LOCALVARIABLE x Ljava/lang/Long; L1 L2 1
    MAXSTACK = 2
    MAXLOCALS = 2
```